.cljfmt/indents.clj

```diff
@@ -1,2 +1,33 @@
-{metabase.api.collection/foo [[:inner 0]]
- toucan.db/insert!           [[:block 1]]}
+;; https://github.com/weavejester/cljfmt/blob/master/cljfmt/resources/cljfmt/indents/clojure.clj
+{auto-parse                                 [[:inner 0]]
+ cond->                                     [[:inner 0]]
+ check                                      [[:inner 0]]
+ deferred-tru                               [[:inner 0]]
+ defprotocol+                               [[:block 1] [:inner 1]]
+ ;ex-info                                 [[:inner 0]]
+ let-404                                    [[:block 1]]
+ mbql.u/match-one                           [[:inner 0]]
+ merge-with                                 [[:inner 0]]
+ mt/dataset                                 [[:inner 0]]
+ mt/formatted-rows                          [[:inner 0]]
+ mt/mbql-query                              [[:inner 0]]
+ mt/test-drivers                            [[:inner 0]]
+ mt/with-temp*                              [[:block 1]]
+ mt/with-temporary-setting-values           [[:block 1]]
+ perms/set-has-full-permissions-for-set?    [[:inner 0]]
+ perms/set-has-full-permissions?            [[:inner 0]]
+ perms/set-has-partial-permissions?         [[:inner 0]]
+ perms/set-has-partial-permissions-for-set? [[:inner 0]]
+ publish-event!                             [[:block 1]]
+ qp.streaming/streaming-response            [[:inner 0]]
+ ;; Do we want this or not?
+ ;revision/revert!                        [[:inner 0]]
+ ;t2/select                                  [[:block 1]]
+ ;t2/select-one                              [[:block 1]]
+ t2/select-one-fn                           [[:inner 0]]
+ t2/select-pk->fn                           [[:inner 0]]
+ t2/update!                                 [[:inner 0]]
```

Comment:
These are wrong, these should all be formatted like normal functions

Reply:
Fixed